tests/test.py

```diff
@@ -1,4 +1,18 @@
 from pprint import pprint
-from trav import Travis
+import pytest
+from trav import Travis, get_svg_from_badge_url, get_travis_badge_url, TRAVIS_BASE_URL, SvgRequestFailed
+
 
 pprint(vars(Travis))
```

Comment:
You might move this to a new `tests/vars.py` file and add `python tests/vars.py` to the Makefile. With pytest, this doesn't get output, and it's nice to be able to see Trav working in the logs.